framework/src/media/audio/audio_manager.c

```diff
@@ -615,29 +617,27 @@ audio_manager_result_t audio_manager_init(void)
 	audio_manager_result_t ret;
 	static int am_initialized = 0;
 	int card_id;
-
+	int found_card = 0;
```

Comment:
I think we can add an empty line after variable declaration. 
Also, after any if or while blocks, its better to have empty line for better readability.